css/index.css

```diff
@@ -1,9 +1,223 @@
-/* Example of @import use
-This imports the `reset.css` file */
-@import "./reset.css";
-
-/* For testing purposes, delete this code */
-body {
-  font-size: max(20vw, 400px);
-  width: min(20vw, 400px);
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.primary-bg {
+  color: white;
+  background-color: yellow;
+}
+
+.edit-me {
+  background-color: #1f2937;
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header-container {
+  width: 80px;
+  height: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-nav-container {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.header-nav-container li {
```

Comment:
We should also do away with compound styling, we can just give the li class and style the class instead of just putting the li here